### PR TITLE
[dom] Upgrade dummy packages

### PR DESCRIPTION
**Pull request by kraushm**

### CODE REVIEW

File: jenkins-builds/7.0.UP00-19.03-gpu [hunk 1]
```diff
@@ -0,0 +1 @@
+ CMake-3.14.1.eb
```
Comment (jgphpc) [inline jenkins-builds/7.0.UP00-19.03-gpu]: also in mc ?

File: easybuild/easyconfigs/m/M4/M4-1.4.18.eb [hunk 2]
```diff
@@ -4,9 +4,13 @@ name = 'M4'
 version = '1.4.18'
 
 homepage = 'http://www.gnu.org/software/m4/m4.html'
-description = """GNU M4 is an implementation of the traditional Unix macro processor. It is mostly SVR4 compatible
-  although it has some extensions (for example, handling more than 9 positional parameters to macros).
- GNU M4 also has built-in functions for including files, running shell commands, doing arithmetic, etc."""
+description = """
+    GNU M4 is an implementation of the traditional Unix macro processor. It
```
Comment (jgphpc) [inline easybuild/easyconfigs/m/M4/M4-1.4.18.eb]: remove end of line trailing whitespace

File: easybuild/easyconfigs/a/Autoconf/Autoconf-2.69.eb [hunk 3]
```diff
@@ -18,8 +21,9 @@ sources = [SOURCELOWER_TAR_GZ]
 dependencies = [('M4', '1.4.18')]
 
 sanity_check_paths = {
-    'files': ["bin/%s" % x for x in ["autoconf", "autoheader", "autom4te", "autoreconf", "autoscan",
-                                     "autoupdate", "ifnames"]],
+    'files': ["bin/%s" % x for x in ["autoconf", "autoheader", "autom4te",
```
Comment (jgphpc) [inline easybuild/easyconfigs/a/Autoconf/Autoconf-2.69.eb]: trailing whitespace

File: easybuild/easyconfigs/l/libtool/libtool-2.4.6.eb [hunk 4]
```diff
@@ -4,8 +4,11 @@ name = 'libtool'
...
 homepage = 'http://www.gnu.org/software/libtool'
-description = """GNU libtool is a generic library support script. Libtool hides the complexity of using shared libraries
- behind a consistent, portable interface."""
+description = """
+    GNU libtool is a generic library support script. Libtool hides the 
+    complexity of using shared libraries behind a consistent, portable 
+    interface.
+"""
 
 toolchain = {'name': 'dummy', 'version': 'dummy'}
```
Comment (jgphpc) [inline easybuild/easyconfigs/l/libtool/libtool-2.4.6.eb]: toolchain = {'name': 'dummy', 'version': `''`}

File: easybuild/easyconfigs/b/bzip2/bzip2-1.0.6.eb [hunk 5]
```diff
@@ -3,9 +3,12 @@ name = 'bzip2'
...
 homepage = 'http://www.bzip.org/'
-description = """bzip2 is a freely available, patent free, high-quality data compressor. It typically 
-compresses files to within 10% to 15% of the best available techniques (the PPM family of statistical 
-compressors), whilst being around twice as fast at compression and six times faster at decompression."""
+description = """
+    bzip2 is a freely available, patent free, high-quality data compressor. It 
+    typically compresses files to within 10% to 15% of the best available 
+    techniques (the PPM family of statistical compressors), whilst being around 
+    twice as fast at compression and six times faster at decompression.
+"""
```
Comment (jgphpc) [inline easybuild/easyconfigs/b/bzip2/bzip2-1.0.6.eb]: move line 18 (`import os`) to the top ?

Comment (jgphpc) [reply easybuild/easyconfigs/b/bzip2/bzip2-1.0.6.eb]: split line 25 ?

File: easybuild/easyconfigs/f/flex/flex-2.6.4.eb [hunk 6]
```diff
@@ -5,18 +5,22 @@ name = 'flex'
...
+]
+
 builddependencies = [
     ('Bison', '3.0.5'),
 ]
 
-source_urls = ['https://github.com/westes/%(name)s/releases/download/v%(version)s']
-sources = [SOURCELOWER_TAR_GZ]
-
 moduleclass = 'lang'
```
Comment (jgphpc) [inline easybuild/easyconfigs/f/flex/flex-2.6.4.eb]: remove blank line

File: easybuild/easyconfigs/f/flex/flex-2.6.4.eb [hunk 7]
```diff
@@ -5,18 +5,22 @@ name = 'flex'
...
 
 toolchain = {'name': 'dummy', 'version': ''}
 
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [
+    'https://github.com/westes/%(name)s/releases/download/v%(version)s'
+]
+
 builddependencies = [
     ('Bison', '3.0.5'),
```
Comment (jgphpc) [inline easybuild/easyconfigs/f/flex/flex-2.6.4.eb]: will it build with Bison/3.3.2 ?

File: easybuild/easyconfigs/z/zlib/zlib-1.2.11.eb [hunk 8]
```diff
@@ -5,18 +5,24 @@ name = 'zlib'
 version = '1.2.11'
 
 homepage = 'http://www.zlib.net/'
-description = """zlib is designed to be a free, general-purpose, legally unencumbered -- that is,
- not covered by any patents -- lossless data-compression library for use on virtually any
- computer hardware and operating system."""
+description = """
```
Comment (jgphpc) [inline easybuild/easyconfigs/z/zlib/zlib-1.2.11.eb]: There is a weird char on line 1:
> cat -v zlib-1.2.11.eb

```
#M-BM- contributed by ...
```

Comment (kraushm) [reply easybuild/easyconfigs/z/zlib/zlib-1.2.11.eb]: Should be fixed.

**Comment by kraushm** [comment]
retest this please

**Pull request merged**